apps/app/src/app/globals.css

```
@@ -11,104 +11,211 @@
+        transparent
+      )
+      transparent;
   }
 }
 
-/* Użyteczne utilsy: glass i gładkie hover cienie dla kart/przycisków */
 @layer utilities {
   .glass {
-    @apply bg-white/60 dark:bg-white/5 backdrop-blur-xl border border-white/20 dark:border-white/10 shadow-sm;
+    @apply border shadow-sm backdrop-blur-xl;
+    background: color-mix(in oklch, var(--card) 72%, transparent);
```

Stylelint (17.11.1)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)